### PR TITLE
Create ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by bode

Ticket: [DM-3282](https://cumulocity.atlassian.net/browse/DM-3282)
Original PR: [6078](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6078)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Extend the "Dynamic forms" page in Codex
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: codex: extend dynamic forms page with additional information
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3282
+version: 1020.0.0
+---
+Extend the "Dynamic forms" page in Codex
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md:17]: The Claude AI suggested to use the following body:
```suggestion
The Codex documentation provides information about developing applications with the Cumulocity IoT platform. To make it easier for users to understand and use dynamic forms in their applications, the "Dynamic forms" page in the Codex documentation has been extended. The page now includes additional information and examples on how to define, style and submit dynamic forms. This change improves the documentation and makes it easier for users to build applications using dynamic forms by providing them with more comprehensive information.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3282
+version: 1020.0.0
+---
+The Codex documentation provides information about developing applications with the Cumulocity IoT platform. To make it easier for users to understand and use dynamic forms in their applications, the **Dynamic forms** page in the Codex documentation has been extended.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md:17]:
```suggestion
The Codex documentation provides information about developing applications with the Cumulocity IoT platform. To make it easier for users to understand and use dynamic forms in their applications, the **Dynamic forms** page in the Codex documentation has been extended. For details, refer to the [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/).
```

@bozhidardedov Can you please provide the exact URL?

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3282
+version: 1020.0.0
+---
+The Codex documentation provides information about developing applications with the Cumulocity IoT platform. To make it easier for users to understand and use dynamic forms in their applications, the **Dynamic forms** page in the Codex documentation has been extended. For details, refer to the [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/).
```
Comment (bozhidardedov) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md:17]:
```suggestion
The Codex documentation provides information about developing applications with the Cumulocity IoT platform. To make it easier for users to understand and use dynamic forms in their applications, the **Dynamic forms** page in the Codex documentation has been extended. For details, refer to the [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/components/forms/dynamic-forms/overview).
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md (hunk 3 above)
Comment (bozhidardedov) [reply content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md]: Commited a change with the full URL after your suggestion.

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Extended information on dynamic forms in Codex
+product_area: Application enablement & solutions
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md:7]: one more thing: I would not call it a feature from a user perspective but an improvement.

Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md:7]:
```suggestion
    label: Improvement
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Extended information on dynamic forms in Codex
+product_area: Application enablement & solutions
+change_type:
+  - value: change-QHu1GdukP
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-extend-the-dynamic-forms-page-in-codex.md:6]:
```suggestion
  - value: change-2c7RdTdXo4
```

**Comment by BeateRixen** [comment]
@bozhidardedov Please approve

**Review by rahultsag — APPROVED** [review]

**Pull request merged**